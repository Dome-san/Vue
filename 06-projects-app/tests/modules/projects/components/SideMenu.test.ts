describe('<SideMenu/>', () => {
  test('should render with no projects', () => {});
});
