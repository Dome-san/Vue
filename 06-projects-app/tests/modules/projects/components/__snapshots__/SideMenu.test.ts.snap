// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<SideMenu/> > should render with no projects 1`] = `
"<aside class="bg-base-200 w-72 min-h-screen">
  <h2 class="text-lg font-bold mx-4">
    <routerlink to="/"> Proyecto</routerlink>
  </h2><!-- Menu -->
  <ul class="menu">
    <li>
      <details>
        <summary>
          <routerlink to="/project/1">Project 1</routerlink>
        </summary>
        <ul>
          <li>
            <routerlink to="/project/1">task 1</routerlink>
          </li>
          <li>
            <routerlink to="/project/1">task 2</routerlink>
          </li>
          <li>
            <routerlink to="/project/1">task 3</routerlink>
          </li>
          <li>
            <routerlink to="/project/1">task 4</routerlink>
          </li>
        </ul>
      </details>
    </li>
    <li>
      <routerlink to="/project/2">Project 2</routerlink>
    </li>
    <li>
      <details>
        <summary>
          <routerlink to="/project/3">Project 3</routerlink>
        </summary>
        <ul>
          <li>
            <routerlink to="/project/3">task 1</routerlink>
          </li>
          <li>
            <routerlink to="/project/3">task 2</routerlink>
          </li>
        </ul>
      </details>
    </li>
    <li>
      <details>
        <summary>
          <routerlink to="/project/4">Project 4</routerlink>
        </summary>
        <ul>
          <li>
            <routerlink to="/project/4">task 1</routerlink>
          </li>
          <li>
            <routerlink to="/project/4">task 2</routerlink>
          </li>
          <li>
            <routerlink to="/project/4">task 3</routerlink>
          </li>
        </ul>
      </details>
    </li>
  </ul>
</aside>"
`;
